src/web-map.js

```diff
@@ -555,15 +542,15 @@ export class WebMap extends HTMLMapElement {
   }
   
   _changeWidth(width) {
-    this.style.width = width+"px";
     this._container.style.width = width+"px";
+    document.querySelector('[is="web-map"]').style.width = width+"px";
```

Comment:
This only selects the 1st map is=web-map in the document, which isn't what you want here, I don't believe.